docs/tutorial/Split_Learning_for_bank_marketing.ipynb

```diff
@@ -110,6 +116,7 @@
     "%autoreload 2\n",
     "\n",
     "import secretflow as sf\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
@zhangxingmeng Can you take a look at this tutorial?

Cheers

Reply:
The PR of  docs/tutorial/Split_Learning_for_bank_marketing.ipynb was proposed in https://github.com/secretflow/secretflow/pull/426.
Now, I have remove the file from this PR.
Please TBR. @anakinxc  Thank you.